src/azul/service/avro_pfb.py

```diff
@@ -384,9 +388,22 @@ def _avro_pfb_schema(azul_avro_schema: Iterable[JSON]) -> JSON:
     null_int: ['string', 'long'],
     null_str: ['string'],
     null_int_sum_sort: ['string', 'long']
+    # null_datetime is handled with a custom logical type 'avro_datetime'
 }
 
 
+def datetime_to_string(data, *args):
+    return format_dcp2_datetime(data)
+
+
+# Fastavro doesn't support `datetime` objects, so when serialized the value is
+# converted to a string via the custom logical type 'avro_datetime'. When the
+# value is deserialized however, we keep it as a string and do not convert it
```

Comment:
```suggestion
# value is deserialized however, we keep it as a string and do not convert it
```

When do we ever deserialize anything while generating the PFB?

Probably better discussed at PL.